src/Mimi.jl

```diff
@@ -721,6 +722,38 @@ end
 
 #End of graph section
 
-include("references.jl")
+# Begin plotting section
+
+"""
+Three defaults: single line plot, multiple line plots with legend, bar graph
+when indexing over regions, etc.
+"""
+function auto_plot(m::Model, component::Symbol, parameter::Symbol ; index::Symbol = :time, legend::Symbol = nothing, xlabel = string(index), ylabel = string(parameter), dest = nothing)
+  if isnull(m.mi)
+      run(m)
+  end
+
+  pyplot()
+  plt = plot() # Clear out any previous plots
+
+  if is(legend, nothing)
+    # Assume that we are only plotting one line (i.e. it's not split up by regions)
+    plot(plt, m.indices_values[index], m[component, parameter])
+  else
+    # For multiple lines, we need to read the legend labels from legend
+    for line_index in 1:size(m[component, parameter])[2] # TODO: Check that these dimensions match
+      plot!(plt, m.indices_values[index], m[component, parameter][:,line_index], label = m.indices_values[legend][line_index])
+    end
+  end
+
+  if !is(dest, nothing)
+    savefig(dest)
+  end
```

Comment:
I think we might not need this, because folks can probably just call something like ``savefig(plt, dest)`` on the value that is returned from this function?